pkgs/development/python-modules/bitmath/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy36, progressbar, mock }:
+
+buildPythonPackage rec {
+  pname = "bitmath";
+  version = "1.3.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1k8d1wmxqjc8cqzaixpxf45k6dl1kqhblr0g4wyjl0qa18q8wasd";
+  };
+
+  doCheck = ! isPy36; # progressbar is incompatible with Py3?
```

Comment:
Not necessary because progressbar is now disabled on python3.

Reply:
I believe it's still necessary, because this package's tests expect progressbar, so if it's disabled on Py3 the tests will fail.

Reply:
It won't even be built on Python 3, so no tests will be run. Just rebase on staging and you'll see what I mean.

Reply:
In fact it doesn't even get to failing the tests — it will try to load `progressbar` for `checkInputs` and immediately fail.

Reply:
That's what I meant with "it won't be built". Please just remove that `doCheck = false` line.

Reply:
But the package itself works just fine with Python 3.6.  Only the test suite fails.

Reply:
I see. Looking at https://github.com/tbielawa/bitmath/blob/master/requirements.txt and https://github.com/tbielawa/bitmath/blob/master/requirements-py3.txt, it seems like `progressbar231` is required for Python 2 and `progressbar33` for Python 3. At least the latter should be packaged so that tests can be enabled for Python 3.
Btw, please use `isPy3k` instead of `isPy36`.

Reply:
It's because `progressbar` has been abandoned by its maintainer. Other people created newer versions, which diverged from each other.

Reply:
Well, I've packaged them up anyway.

Reply:
Thanks a lot!